main.go

```diff
@@ -0,0 +1,28 @@
+package main
```

Comment:
> I guess the other question I have is why do you think this should only be an example? This package (server/) is implementing a service that should compile to an executable. The reusable components have already been packaged elsewhere.

I added the description into the README. Maybe I'm misunderstanding here. I thought that the `server` package is still a CONIKS library which implements the registration, key lookup functions... Then the developers can add some extra layers on the top of the library, e.g. storage backend which stores data that not in the tree, or something else ... Or, they can choose to whether they want to use the registration bots library. 

> Stated another way, why are we asking the ops team to implement main.go? If anything, I would double down on this and move -mkkey and -mkconfig as flags here.

Yes. If it is an executable, I will do it.